CRM/Core/BAO/CustomValueTable.php

```diff
@@ -48,8 +48,7 @@ public static function create($customParams, $parentOperation = NULL) {
         $count = 1;
 
         $firstField = reset($fields);
-        $entityID = $firstField['entity_id'];
```

Comment:
The brevity of casting to a var vs the loss of clarify is a little greater for entity_id as entityID is less vague & it's used more